metaci/plan/handlers.py

```diff
@@ -0,0 +1,16 @@
+from django.dispatch import receiver
+from metaci.build.signals import build_complete
+from metaci.plan.models import PlanRepositoryTrigger
+
+
+@receiver(build_complete)
+def trigger_dependent_builds(sender, **kwargs):
+    build = kwargs.get("build")
+    status = kwargs.get("status")
```

Comment:
Any reason not to just make the function signature be `(sender, build, status)`?

Reply:
My understanding is that it's less tightly coupled to the `build_complete` signal's arguments. ([docs](https://docs.djangoproject.com/en/2.1/topics/signals/#receiver-functions))